### PR TITLE
CONSOLE-3393: Proxy managed cluster monitoring requests through MCE c…

### PR DESCRIPTION
**Pull request by TheRealJon**
…luster proxy

We still need to add a ManagedServiceProxyResolver to the console operator manifest so that MCE knows how to resolve these requests.

### CODE REVIEW

**Comment by kyoto** [comment]
CC @openshift/openshift-team-monitoring

**Review by jhadvig — APPROVED** [review]
/lgtm

**Comment by yapei** [comment]
- Hub Cluster metrics pages and data are showing correctly
- Managed clusters metrics pages and data are showing successfully
- managedproxyserviceresolver is created
no issues found
/label qe-approved

**Comment by yapei** [comment]
sorry, it seems we have a regression issue [regression issue](https://issues.redhat.com/browse/OCPBUGS-7041) which causes problems when normal user view metrics, could you help check?  
/remove-qe-approved

**Comment by TheRealJon** [comment]
@yapei I've made some changes in this PR and was not able to reproduce the above regression. Do you mind testing again?

**Comment by yapei** [comment]
@TheRealJon  tested again and above regression is not reproduced, however I see some warning info when viewing Alerting on managed cluster
managed-cluster-error-loading-silences

**Comment by TheRealJon** [comment]
Hi @yapei! That is expected right now. We don't have alertmanager service proxies set up yet. That will be a separate issue.

**Comment by yapei** [comment]
based on testing result and the discussions above, everything works as expected
/label qe-approved

**Review by jhadvig — APPROVED** [review]
Nice work @TheRealJon 👍 
/lgtm

File: pkg/server/server.go [hunk 1]
```diff
@@ -41,162 +41,169 @@ import (
...
-	QuickStarts                  string
-	AddPage                      string
-	ProjectAccessClusterRoles    string
-	Perspectives                 string
-	Telemetry                    serverconfig.MultiKeyValue
-	CopiedCSVsDisabled           map[string]bool
-	HubConsoleURL                *url.URL
-
-	// TODO: only needed because handleLogoutMulticluster is part of the server type
-	AuthMetrics *auth.Metrics
```
Comment (TheRealJon) [inline pkg/server/server.go:193]: Everything above here is just housekeeping. Alphabetized struct properties and constants.

**Comment by TheRealJon** [comment]
@jhadvig Had to rebase. PTAL again, thanks!

**Review by jhadvig — APPROVED** [review]
/lgtm

**Pull request merged**